lockbox-ios/View/SettingsView.swift

```diff
@@ -149,3 +164,14 @@ class SwitchSettingCellConfiguration: SettingCellConfiguration {
         self.isOn = isOn
     }
 }
+
+class CheckmarkSettingCellConfiguration: SettingCellConfiguration {
+    var isChecked: Bool = false
+    var value: Any?
```

Comment:
`value` is not a descriptive variable name. I have no idea what that property is for.